scripts/figure_setup/sync-data-files.R

```diff
@@ -63,3 +63,13 @@ fs::dir_create(celltype_local_files)
 
 sync_call <- glue::glue("aws s3 cp '{celltype_s3_files}' '{celltype_local_files}' --exclude '*' --include 'SCPCL000498_processed.rds' --recursive")
 system(sync_call)
+
+# sync merged object results for SCPCP000003 -----------------------------------
```

Comment:
I wonder if generally we should update this script to use `aws s3 sync` instead of `aws s3 cp`? It's my understanding (am I correct?) that `cp` will _always_ copy, but `sync` will only copy if the local file doesn't exist or is out of date.

Reply:
Also, I think some of the other syntax (like `--recursive`) would change?

Reply:
Just wanted to note that I've been burned before by `sync`, because it doesn't always update if the file is out of date. I had this issue when benchmarking alevin-fry, which is why I've been using `cp` for all of this.

Reply:
Maybe workaround with `if` then? Only `cp` if the target files don't exist?

Reply:
Though I guess then we won't get new files if things are out of date. I wonder if there are _some_ files this would help for then, if not all.

Reply:
I think we generally want to make sure we are copying over the up to date files. The only files that I imagine will never change, mainly because we deleted the FASTQ files, are the alevin-fry and cell ranger output. So we could add an `if` statement around those, but I also don't know if that's totally necessary.

Reply:
> To be fair, my comment was brought to you by, "ugh so much copying until i get to copy the file i want to test the code"

This is a valid point. We could break up that script or add options so it only copies what you need? So a flag for each of the items that need to be copied and only copy them if that flag is used?

Reply:
Maybe... The best idea I can think of for flags is a figure name? So like, `--fig_3` will copy files needed for that figure only, and `--all` would do everything. But, this also seems like a not great use of time considering the code works, and I am capable of doing some quick script changes when I only want _some_ files! Plus, this is probably only relevant to running code during development.

Maybe the right approach is to have a comment up top: This will always copy all files regardless of whether they are on your system already, and justt move on.

Reply:
How often are these files changing? This is just for the paper, right? I'd probably switch to `sync` and then occasionally delete files to make sure they are up to date. Note thought that `sync` is for directories, so individual file downloads might have to be handled differently (with `--include`)

I also want to believe that the earlier problems with `sync` are in the past and we can trust it again. If we want to be more conservative, we can also use `--exact-timestamps` which will download the file even if the local copy is newer (if it was `touch`ed somehow for example). Finally, before any publication runs, you could delete the local files to force a new download.

Reply:
Based on this discussion, I went ahead and moved everything to be `sync` with `--exact-timestamps` rather than `cp`. I also updated the script for generating figures to include running the scripts to download the data before generating the figures. I added a step to remove `s3_files` before running the download scripts so that everything is fresh. These changes are in https://github.com/AlexsLemonade/scpca-paper-figures/pull/58/commits/a295eadec9a25f333b2392c9669a7896cd7a39d1 and https://github.com/AlexsLemonade/scpca-paper-figures/pull/58/commits/3bd8250e1b137401f2d7b374bb14fc5e15e0320b